test/Microsoft.DotNet.Watcher.Tools.Tests/ArgumentSeparatorTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.DotNet.Watcher.Tools.Tests
+{
+    public class ArgumentSeparatorTests
+    {
+        [Theory]
+        [InlineData(new string[0],
```

Comment:
We cannot test that in a unit test. The quotation for arguments happens in a different part of the code